<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::TabPosition::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::TabShape::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>isimsiz</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTextEdit" name="textEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>isimsiz</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAra_lar">
    <property name="title">
     <string>Araçlar</string>
    </property>
    <addaction name="actionOpenFile"/>
    <addaction name="actionSave"/>
    <addaction name="actionNewFile"/>
   </widget>
   <addaction name="menuAra_lar"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenFile">
   <property name="text">
    <string>Dosya aç</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Kaydet</string>
   </property>
  </action>
  <action name="actionNewFile">
   <property name="text">
    <string>Yeni Dosya Oluştur</string>
   </property>
  </action>
  <action name="actionEditFile">
   <property name="text">
    <string>edit_file</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
